### ISSUE TITLE
[FEATURE] I'd like to see the memory a given execution consumes

### ISSUE THREAD

**Issue by dylburger** [PipedreamHQ/pipedream#5329]
I'd like to see exactly how much memory was used by my workflow.

**Comment by rrbbtalnorth** [comment]
I agree to the request

**Comment by krypston104** [comment]
yes, please. A tool like this is needed to tune it and avoid out of memory errors

**Comment by eligeske** [comment]
🪦  Arise!

**Comment by helloitse** [comment]
Seems like an oversight (or intended ux hurdle) to make users responsible for setting memory consumption but not provide any tools to track it 🤷🏾‍♀️
